gtests/net/mptcp/sockopts/sockopt_cork_multi.pkt

```diff
@@ -0,0 +1,50 @@
+--tolerance_usecs=100000
+`../common/defaults.sh`
++0    `../common/multi-ep.sh -e 0`
+
++0.0  socket(..., SOCK_STREAM, IPPROTO_MPTCP) = 3
++0.0  setsockopt(3, SOL_SOCKET, SO_REUSEADDR, [1], 4) = 0
++0.0  setsockopt(3, SOL_TCP, TCP_CORK, [1], 4) = 0
++0.0  fcntl(3, F_GETFL) = 0x2 (flags O_RDWR)
++0.0  fcntl(3, F_SETFL, O_RDWR|O_NONBLOCK) = 0
+
++0    bind(3, ..., ...) = 0
++0    listen(3, 1) = 0
+
++0.0    <  addr[caddr0] > addr[saddr0]  S   0:0(0)               win 65535  <mss 1460, sackOK, TS val 4074410674 ecr 0,          nop, wscale 8, mpcapable v1 flags[flag_h] nokey>
++0.0    >                               S.  0:0(0)        ack 1             <mss 1460, sackOK, TS val 4074410674 ecr 4074410674, nop, wscale 8, mpcapable v1 flags[flag_h] key[skey]>
++0.0    <                                .  1:1(0)        ack 1  win 256    <nop, nop,         TS val 4074410674 ecr 4074410674,                mpcapable v1 flags[flag_h] key[ckey=2, skey]>
++0    accept(3, ..., ...) = 4
+
++0.2    <                               P.  1:3(2)        ack 1  win 256    <nop, nop, TS val 4074418292 ecr 4074410674,                        mpcapable v1 flags[flag_h] key[skey, ckey] mpcdatalen 2, nop, nop>
+// MP_CAPABLE carrying data are acked with 64-bit, safer not knowing if the
+// sender will use a DSN on 64-bit or 32-bit. Later we can use the lower 32 bits
++0.0    >                                .  1:1(0)        ack 3             <nop, nop, TS val 4074418293 ecr 4074418292, dss dack8=3  dll=0 nocs>
++0.0  read(4, ..., 2) = 2
+
+// more inbound data and MP_PRIO to flip active -> backup
```

Comment:
why do you need this?

Reply:
Will be removed, part of C&P from mptcp/mp_prio/mp_prio_server.pkt after I tried flipping active->backup myself and failed. :) Wanted to see setsockopt affecting multiple subflows.